src/FSharp.Editing.VisualStudio/Coloring/UnusedDeclarationMargin.fs

```diff
@@ -1,4 +1,4 @@
-﻿namespace FSharpVSPowerTools
+﻿namespace FSharp.Editing.VisualStudio.Coloring.UnusedSymbols
```

Comment:
Several types names clash with ones from `SymbolClassifier.fs` since the implementations of normal and unused symbol classifiers are similar.